src/ctia/import/capec.clj

```diff
@@ -0,0 +1,164 @@
+(ns ctia.import.capec
+  (:require [clojure.xml :as xml]
+            [ctim.schemas.ttp :as ttp]
+            [schema.core :as s]
+            [ctia.lib.time :as time]
+            [clojure.pprint :refer [pprint]]
+            [ctia.import.threatgrid.http :as http]))
+
+(defn capec-tag [kw]
+  (keyword (str "capec" kw)))
+
+(defn sieve
+  "Helps keep clojure.xml traversal code more concise, with sensible defaults."
+  ([e tag] (sieve e tag first :content))
```

Comment:
Thank you, I'll definitely have a look at that--I asked around for what the idiomatic way to do this was, because it felt very very verbose before.